presto-main/src/test/java/com/facebook/presto/operator/GroupByHashYieldAssertion.java

```diff
@@ -73,7 +73,7 @@ private GroupByHashYieldAssertion() {}
      */
     public static GroupByHashYieldResult finishOperatorWithYieldingGroupByHash(List<Page> input, Type hashKeyType, OperatorFactory operatorFactory, Function<Operator, Integer> getHashCapacity, long additionalMemoryInBytes)
     {
-        assertLessThan(additionalMemoryInBytes, 1L << 20, "additionalMemoryInBytes should be a relatively small number");
+        assertLessThan(additionalMemoryInBytes, 2L << 20, "additionalMemoryInBytes should be a relatively small number");
```

Comment:
Did you mean to make this change?

Reply:
@dain : Yes. Previously the `PageBuilder` in aggregator will hold up to ~100KB memory, now it's ~1MB so we need to tune up this number.

There might be some other way to avoid tune this number.